evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -52,7 +52,7 @@ open class PackageRule(
     private val licenseRules = mutableListOf<LicenseRule>()
 
     @Suppress("UNUSED") // This is intended to be used by rule implementations.
-    val uncuratedPkg: Package? by lazy { ruleSet.ortResult.getUncuratedPackageOrProject(pkg.metadata.id) }
+    val uncuratedPkg: Package by lazy { ruleSet.ortResult.getUncuratedPackageOrProject(pkg.metadata.id)!! }
```

Comment:
A hint in the commit message why we can assume the return value of `getUncuratedPackageOrProject()` to never be `null` here would have been nice.